system-addon/content-src/components/Card/Card.jsx

```diff
@@ -119,7 +119,8 @@ export class Card extends React.PureComponent {
           </div>}
           <div className={`card-details${hasImage ? "" : " no-image"}`}>
             {link.hostname && <div className="card-host-name">{link.hostname}</div>}
-            <div className={["card-text",
+            <div className={[
+              "card-text",
```

Comment:
moved this to a new line to have the closing `]` line up as a block instead of inline